src/OpenConext/ProfileBundle/Security/Authentication/Entity/User.php

```diff
@@ -49,11 +62,6 @@ class User
      */
     public function __toString()
     {
-        return serialize([
-            $this->nameId,
-            $this->institution,
-            $this->email,
-            $this->commonName
-        ]);
+        return serialize($this);
```

Comment:
`User::__toString()` is used for:

0. change detection in `AbstractToken::setUser()`;
0. and display in Symfony's web toolbar.

Perhaps it is good to change this to `'User('.sha1(serialize($this)).')'`. Change detection will still work and the web toolbar doesn't display a logged in user that wraps and covers the entire width of the screen. Or perhaps get the EPTI from the attributes. Opinions, @DRvanR?